File: CS/TreeViewReport/Form1.cs
namespace TreeViewReport {
    public partial class Form1 : DevExpress.XtraBars.Ribbon.RibbonForm {
        TreeReport report;
        public Form1() {
            InitializeComponent();
            report = new TreeReport(this);
            documentViewer1.DocumentSource = report;
            treeList1.KeyFieldName = "Key";
            treeList1.ParentFieldName = "ParentKey";
        }
    }
}

File: VB/TreeViewReport/Form1.vb
Namespace TreeViewReport

    Public Partial Class Form1
        Inherits DevExpress.XtraBars.Ribbon.RibbonForm

        Private report As TreeReport

        Public Sub New()
            InitializeComponent()
            report = New TreeReport(Me)
            documentViewer1.DocumentSource = report
            treeList1.KeyFieldName = "Key"
            treeList1.ParentFieldName = "ParentKey"
        End Sub
    End Class
End Namespace
